src/test/java/htsjdk/samtools/util/BlockCompressedInputStreamTest.java

```diff
@@ -86,4 +92,92 @@ public void available_should_return_number_of_bytes_left_in_current_block() thro
 			}
 		}
 	}
+
+    private static class MyInflater extends Inflater {
+        MyInflater(boolean gzipCompatible) {
+            super(gzipCompatible);
+        }
+        @Override
+        public int inflate(byte[] b, int off, int len) throws java.util.zip.DataFormatException {
+            inflateCalls++;
+            return super.inflate(b, off, len);
+        }
+        static int inflateCalls;
```

Comment:
This should be non-static and initialized at construction time, to avoid the possibility of separate test cases incorrectly sharing the counter.